### PR TITLE
feat : Landing page implemented

### PR DESCRIPTION
**Pull request by govardhanreddyc79**

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice, please add a screenshot or Gif if possible

**Review by saicharan-gouru — APPROVED** [review]
kindly consider the mentioned suggestions.
-UI looks good to me. 
-Please add description and screenshot for reference.
-You can also add deploy preview link.
-The code is clean but you can beautify the code structure(like removing extra spaces etc..)

File: index.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+.homepage-container {
+    width:100vw;
+    height: 80vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.homepage-container p {
```
Comment (saicharan-gouru) [inline index.css:14]: Don't use tag names directly. Instead give a class or id to the element

File: index.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    font-size: 2rem;
+    margin: 0;
+    color: var(--secondary-color);
+}
+.homepage-container h1 {
```
Comment (saicharan-gouru) [inline index.css:19]: Don't use tag names directly. Instead give a class or id to the element

**Pull request merged**